Tests/Resource/ResourceTests.swift

```diff
@@ -24,48 +24,35 @@ extension Todo: ContentMappable {
 extension Todo: StructuredDataRepresentable {
     public var structuredData: StructuredData {
         return [
-            "id": id.map({StructuredData.from($0)}) ?? nil,
-            "title": StructuredData.from(title),
-            "done": StructuredData.from(done)
+            "id": id.map({StructuredData($0)}) ?? nil,
+            "title": StructuredData(title),
+            "done": StructuredData(done)
         ]
     }
 }
 
 class ResourceTests: XCTestCase {
     func testReality() {
```

Comment:
I think we need more tests here. While I have faith in your code, theres a lot of points where it could go wrong (especially with the `Mapper` stuff).